app/domain/authentication/authn_k8s/consts.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Authentication
+  module AuthnK8s
+
+    K8S_AUTHENTICATOR_NAME = 'authn-k8s'
```

Comment:
this can be `AUTHENTICATOR_NAME` because its full name is `Authentication:: AuthnK8s::AUTHENTICATOR_NAME`.

but it's ok as it is if you prefer that.

Reply:
The module is not always there...

Reply:
> The module is not always there...

In these cases it should only work if you are already inside the module, as you are here.  So either context or a fully qualified path should always make the "K8s" part clear, and we should never have a case where we require "name stutter".  This is true in general too.

Reply:
Indeed, just noticing now that the existing convention has stutter too: `Authentication::AuthnK8s` should probably be `Authentication::K8s`, and arguably my original choice of `Authentication` should have been `Authn` so that we'd have `Authn::K8s` as the prefix, which is, in addition to being shorter, much more readable imo.

That parts out of scope though :)

Reply:
done.